src/server/index.js

```diff
@@ -0,0 +1,20 @@
+const express = require("express");
```

Comment:
When we switch to babel we will have to decide if we want to transpile these node.js only files.

I'd lean towards transpiling as little as we need, but I'm open to anything. Let's discuss in person or when we reach that point.